File: 2026-03/src/day_18/boj_1919/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int cnt[26];

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    string s1, s2;
    cin >> s1 >> s2;

    for (char c : s1) cnt[c - 'a']++;
    for (char c : s2) cnt[c - 'a']--;

    int ans = 0;
    for (int x : cnt) {
        ans += abs(x);
    }

    cout << ans;
}

File: 2026-03/src/day_18/boj_1919/Main.java
package day_18.boj_1919;

import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] cnt = new int[26];

        String s1 = br.readLine();
        String s2 = br.readLine();

        for (char c : s1.toCharArray()) {
            cnt[c - 'a']++;
        }
        for (char c : s2.toCharArray()) {
            cnt[c - 'a']--;
        }

        int ans = 0;
        for (int x : cnt) {
            ans += Math.abs(x);
        }

        System.out.println(ans);
    }
}
